idlpy/src/types.c

```diff
@@ -185,42 +229,68 @@ emit_field(
         }
     }
 
-    idlpy_ctx_printf(ctx, "\n    %s: %s", name, type);
+    //Reserved Python keywords support (Issue 105):
+    const char *nameField = name;
+    const char *nameFieldModified=get_c_name(name);
+    if (nameFieldModified)
+        nameField =  nameFieldModified;
+    
+    idlpy_ctx_printf(ctx, "\n    %s: %s", nameField, type);
+
+    if (nameFieldModified) {
```

Comment:
If you switch everything to `idlpy_identifier` this will become `if (name != idl_identifier(node))`. Note that `strcmp` is not needed, we're comparing constant pointers and that is fine in this case.